docs/02-guides/extend-vite-configuration/_category_.yml

```diff
@@ -0,0 +1,7 @@
+label: Extend the Vite Configuration
+collapsible: true
+link:
+  type: generated-index
+  description:
+    This section contains goal-oriented documentation that provides step-by-step
+    instructions to extend the Vite configuration with popular extensions.
```

Comment:
Although this was in the ticket, i'm not sure what this has to do with the vite configuration.

Reply:
The implementation of tailwind is made thanks to vite, i think it's more something like "this things that you can do with front-commerce can be done thanks to vite"

Reply:
Exactly. The goal is to show common use cases for Vite specific configuration.